src/security/ServerOptions.cc

```diff
@@ -351,6 +351,12 @@ Security::ServerOptions::loadClientCaFile()
     return bool(clientCaStack);
 }
 
+/// Interprets DHE parameters stored in a previously configured dhParamsFile.
+/// These DHE parameters are orthogonal to ECDHE curve name that may also be
+/// configured when naming that DHE parameters configuration file. When both are
+/// configured, the server selects either FFDHE or ECDHE key exchange mechanism
+/// (and its cipher suites) depending on client-supported cipher suites.
+/// \sa Security::ServerOptions::updateContextEecdh() and RFC 7919 Section 1.2
 void
 Security::ServerOptions::loadDhParams()
```

Comment:
@sp-andwei, I am done with code adjustments. No primary functionality changes expected compared to your version, but please retest. If you have any questions about the adjustments, please discuss.

Please also check this method description. If the description is accurate, I will fix PR description (i.e. future master commit message body) with this confirmed understanding in mind. If this method description is wrong or can be improved, please commit a new comment version. Thank you!

There is no need to rebase or squash-merge PR branch changes. [Anubis](https://github.com/measurement-factory/anubis/blob/master/README.md), our merge bot, will take care of that when the time comes to stage and merge this PR.

Reply:
I think your description is totally fine.

I applied all your polishing changes and checked again for ECDHE and DHE and both work as expected with a setting of `tls-dh=prime256v1:<path>/dhparams4096.pem` (using Ubuntus 24.04. `openssl s_client`, version 3.0.13)